roles/rutorrent/defaults/main.yml

```diff
@@ -0,0 +1,282 @@
+  RT_RECEIVE_BUFFER_SIZE: "{{ lookup('role_var', '_rt_receive_buffer_size') | default(omit, true) }}"
+  RT_SEND_BUFFER_SIZE: "{{ lookup('role_var', '_rt_send_buffer_size') | default(omit, true) }}"
+  RT_PREALLOCATE_TYPE: "{{ lookup('role_var', '_rt_preallocate_type') | default(omit, true) }}"
+  RT_DHT_PORT: "{{ (rutorrent_dht_port | string) if rutorrent_dht_port != 6881 else omit }}"
+  RT_INC_PORT: "{{ (rutorrent_inc_port | string) if rutorrent_inc_port != 50000 else omit }}"
+  RU_REMOVE_CORE_PLUGINS: "{{ (lookup('role_var', '_trim_plugins_enable', default=true) | bool
+                               | ternary('_cloudflare,autotools,cookies,extsearch,feeds,ipad,loginmgr,
+                                         rss,rssurlrewrite,rutracker_check,unpack', '') | split(',')
+                               | union(lookup('role_var', '_ru_remove_core_plugins', default='') | split(','))
+                               | select | map('trim') | join(',')) | default(omit, true) }}"
+  RU_HTTP_USER_AGENT: "{{ lookup('role_var', '_ru_http_user_agent') | default(omit, true) }}"
+  RU_HTTP_TIME_OUT: "{{ lookup('role_var', '_ru_http_time_out') | default(omit, true) }}"
+  RU_HTTP_USE_GZIP: "{{ lookup('role_var', '_ru_http_use_gzip') | default(omit, true) }}"
+  RU_RPC_TIME_OUT: "{{ lookup('role_var', '_ru_rpc_time_out') | default(omit, true) }}"
+  RU_LOG_RPC_CALLS: "{{ lookup('role_var', '_ru_log_rpc_calls') | default(omit, true) }}"
+  RU_LOG_RPC_FAULTS: "{{ lookup('role_var', '_ru_log_rpc_faults') | default(omit, true) }}"
+  RU_PHP_USE_GZIP: "{{ lookup('role_var', '_ru_php_use_gzip') | default(omit, true) }}"
+  RU_PHP_GZIP_LEVEL: "{{ lookup('role_var', '_ru_php_gzip_level') | default(omit, true) }}"
+  RU_SCHEDULE_RAND: "{{ lookup('role_var', '_ru_schedule_rand') | default(omit, true) }}"
+  RU_LOG_FILE: "{{ lookup('role_var', '_ru_log_file') | default(omit, true) }}"
+  RU_DO_DIAGNOSTIC: "{{ lookup('role_var', '_ru_do_diagnostic') | default(omit, true) }}"
+  RU_CACHED_PLUGIN_LOADING: "{{ lookup('role_var', '_ru_cached_plugin_loading') | default(omit, true) }}"
+  RU_SAVE_UPLOADED_TORRENTS: "{{ lookup('role_var', '_ru_save_uploaded_torrents') | default(omit, true) }}"
+  RU_OVERWRITE_UPLOADED_TORRENTS: "{{ lookup('role_var', '_ru_overwrite_uploaded_torrents') | default(omit, true) }}"
+  RU_FORBID_USER_SETTINGS: "{{ lookup('role_var', '_ru_forbid_user_settings') | default(omit, true) }}"
+  RU_LOCALE: "{{ lookup('role_var', '_ru_locale') | default(omit, true) }}"
+rutorrent_role_docker_envs_custom: {}
+rutorrent_role_docker_envs: "{{ lookup('role_var', '_docker_envs_default')
+                                | combine(lookup('role_var', '_docker_envs_custom')) }}"
+
+# Volumes
+rutorrent_role_docker_volumes_download: false
+rutorrent_role_docker_volumes_default:
+  - "{{ rutorrent_role_paths_location }}:/config"
+  - "{{ lookup('role_var', '_topdir_path', default=rutorrent_role_paths_downloads_location) }}:/data:ro"
```

Comment:
read only?

Reply:
Nothing uses `/data` since `TOPDIR_PATH` and `DOWNLOAD_PATH` are both set to /mnt paths, but without this mount the image creates an anonymous volume for it (same with `/passwd`). ro prevents the init script creating unused directories in there, which I noticed happens on occasion.